frontend/src/lib-components/molecules/date-picker/DatePickerInput.tsx

```diff
@@ -37,26 +36,21 @@ const DateInputField = styled(InputField)`
 `
 
 export default React.memo(function DatePickerInput(props: Props) {
-  const i18n = useTranslations()
   const { locale, useBrowserPicker, ...rest } = props
-  const ariaId = useUniqueId('date-picker-input')
 
   return (
     <>
       {useBrowserPicker ? (
-        <DateInputNative {...rest} locale={locale} ariaId={ariaId} />
+        <DateInputNative locale={locale} {...rest} />
       ) : (
-        <DateInputText {...rest} locale={locale} ariaId={ariaId} />
+        <DateInputText locale={locale} {...rest} />
       )}
-      <HelpTextForScreenReader lang={locale} id={ariaId}>
-        {i18n.datePicker.description}
-      </HelpTextForScreenReader>
     </>
   )
 })
 
 interface InternalProps extends Omit<Props, 'useBrowserPicker'> {
-  ariaId: string
+  ariaId?: string
```

Comment:
is this prop used at all anymore? if not, I think it should be removed

Reply:
updated, removed